kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -24,10 +24,17 @@ const publicMethods = [
  * is configured in webpack.config.js
  * @classdesc
  */
-function Lib() {
+function Lib(kolibriGlobal) {
   // libraries
   this.loglevel = require('loglevel');
   this.vue = require('vue');
+  this.vue.mixin({
+    methods: {
+      url: (url_name, ...args) => {
+        return kolibriGlobal.urls[url_name](...args);
+      }
```

Comment:
Here are some examples in KA Lite, where we also used Django-js-reverse: https://github.com/learningequality/ka-lite/search?l=javascript&q=%22urls.%22&type=Code&utf8=%E2%9C%93